includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -354,6 +354,7 @@ public function __construct(
 		add_action( 'woocommerce_update_options_payment_gateways_' . $this->id, [ $this, 'process_admin_options' ] );
 		add_action( 'admin_notices', [ $this, 'display_errors' ], 9999 );
 		add_action( 'woocommerce_woocommerce_payments_admin_notices', [ $this, 'display_test_mode_notice' ] );
+		add_action( 'woocommerce_woocommerce_payments_admin_notices', [ $this, 'display_not_supported_currency_notice' ] );
```

Comment:
It may be better to add this notice to the admin as a whole and not just the payments page. Presently if they change the currency after setting up the gateway, they won't see the notice and their store will be broken.

Reply:
Thanks a lot for the suggestion! 😸 Totally on point 👍 

Changed in https://github.com/Automattic/woocommerce-payments/pull/2558/commits/36c42a038a07a152c29bb1706da2731d18d72e3e